raceSystem/webUI.py

```diff
@@ -0,0 +1,194 @@
+from flask import Flask, render_template, request, jsonify
+from raceSystem.userInterface import UserInterface
+
+
+class WebIU(UserInterface):
+    def __init__(self) -> None:
+        self._app = Flask(__name__)
+        self._startClientConnected = False
+        self._finishClientConnected = False
+        self._raceStatus = ''
+        self._raceResult = ['', '']
+
+    def run(self):
+        """Start the user interface"""
+        self.setupFlaskRouting()
+        self._app.run(debug=True, use_reloader=False)
+
+    def setupFlaskRouting(self):
+        self.homePage()
+        self.formRaceControl()
+        self.formNames()
+        self.settingsPage()
+        self.statusRequest()
+
+    def homePage(self):
+        @self._app.route("/", methods=['GET', 'POST'])
+        def index():
+            name1 = self.getNameCallback(0)
+            name2 = self.getNameCallback(1)
+            return render_template(
+                "home.html",
+                name_1=name1 + " ",
+                name_2=name2 + " ")
+
+    def formRaceControl(self):
+        @self._app.route("/raceControl", methods=['POST'])
+        def raceControl():
+            if request.method == 'POST':
+                if request.form.get('btnStart') == 'Start':
+                    self.startRaceCallback()
+                elif request.form.get('btnStop') == 'Stop':
+                    self.stopRaceCallback()
+            return '', 204
+
+    def formNames(self):
```

Comment:
I get an error when I try to save new names

Reply:
Fixed.